test/test_sync_subclassof.py

```diff
@@ -0,0 +1,34 @@
+"""Tests for sync_subclassof.py"""
+import os
+import sys
+import unittest
+from pathlib import Path
+
+TEST_DIR = Path(os.path.abspath(os.path.dirname(__file__)))
+TEST_INPUT_DIR = TEST_DIR / 'input'
+TEST_OUTPUT_DIR = TEST_DIR / 'output'
+PROJECT_ROOT = TEST_DIR.parent
+sys.path.insert(0, str(PROJECT_ROOT))
+TEST_ONTOLOGY = TEST_INPUT_DIR / 'merged.owl'
```

Comment:
## Testing

Reply:
This needs thought beyond my capacity. in my view, we should create a fork of the mondo ingest with a small set of test ontologies and develop on the fork, and when everything works as expected, merge into the parent project. I can see another way right now, because of the way ODK/Makefiles and so on are convoluted together..

Reply:
@matentzn I think how you're thinking about this is much more holistic and will take a lot more time than my as-needed approach. Not to say it's wrong. Because of your being highly busy it may be that I could see myself as being responsible with coming up with a solution for my current problem as well as the broader objective you mention.

Created an issue here:
- #373

Linked to the grouping issue:
- #318

Reply:
Ok, feel free to pitch it, I don't know how dependent the infrastructure is on ODK stuff..

Reply:
I'm not necessarily pitching the ad hoc approach per se, but the approach is:
- I create a small OWL file, in an ad hoc way, just for the purposes of the tests I make for this PR only

Your approach I think is mostly:
- Come up with a good algorithmic way (with some config) to create tiny versions of every OWL and SemanticSQL file used in the build pipeline
- Have a "test build" process that runs a miniature build using these files and runs test